public/views/index.html

```diff
@@ -1,257 +1,194 @@
-<script type="text/javascript">
-$(document).ready(function() {
+<header class="main-header">
+  <div class="h-container">
 
-  var ChroMo = '(Chrome)/(\d+)\.(\d+)\.(\d+)\.(\d+) Mobile';
-
-  if (/Android|webOS|CriOs|ChroMo|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent) === false && location.href !== "http://localhost:3000/#!/app") {
-    $('body').append('<a href="http://hackreactor.com"><img style="z-index: 10000; position: absolute; top: 0; right: 0; width: 125px; height: 125px; border: 0;" src="http://i.imgur.com/x86kKmF.png" alt="Built at Hack Reactor"></a>');
-  }
-
-  //Twitter buttons
-  !function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');
-
-  });
-</script>
+    <div class="logo">
+      <img src="/img/playing-cards.png" alt="">
+      <h5>Cards for Humanity</h5>
+    </div>
 
-<script></script>
-<div landing id="main-bod">
-  <!-- Fixed Logo & Menu Bar Start -->
-  <div id="top-part">
-    <div class="wpb-top-bar">
-      <div class="mod-contain" id="navver">
-        <div class="row">
-          <!-- Logo -->
-          <div class="wpb-logo-container">
-            <div id="landing-logo">
-              <span id="first-word">Cards</span>
-              <span id="second-word"> for</span>
-              <span id="third-word">Humanity</span>
-            </div>
-            <div class="col-md-9 wpb-menu-container">
-              <ul class="wpb-main-menu">
-                <li><a onClick="scroll(0,0)">Home</a></li>
-                <li><a onClick="scroll(0,742);">How to Play</a></li>
-                <li><a onClick="scroll(0,1411);">Charity</a></li>
-                <li><a onClick="scroll(0,2021);">About Us</a></li>
-                <span ng-show="showOptions">
-                  <li><a href="#!/signup">Sign Up</a></li>
-                  <li><a href="#!/signin">Sign In</a></li>
-                </span>
-                <span ng-hide="showOptions">
-                  <li><a onclick="logout()">Sign Out</a></li>
-                </span>
-              </ul>
-            </div>
-          </div>
-        </div>
-      </div>
+    <nav class="site-nav" id="nav-id">
+      <ul>
+        <li>
+          <a href="">Home</a>
+        </li>
+        <li>
+          <a href="">
+            </i>Meet the Team</a>
+        </li>
+        <li>
+          <a href="">Charity</a>
```

Comment:
Considering this is a one page application, with the way these links are setup, will they scroll to the sections of  the page when clicked on? Please confirm that.

Reply:
For now no. But I will ensure it works that way on my next PR